contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -73,18 +73,21 @@ <h4>Contribution long description</h4>
     {% if post.contributors %}
         <h4>Contributor details</h4>
         {% for c in post.contributors %}
+            <h5><b>{{ c["contributorType"] }}</b></h5>
             {% for key,val in c.items() %}
                 {% if key == "affiliation" %}
-                    {% for k,v in val.items() %}
-                        {% if v %}
+                    {% for k,vk in val.items() %}
+                        {% if vk %}
                         <div class="row">
                             <div class="col-12 col-sm-2">
-                                {{k}}
+                                affiliation {{k}}
```

Comment:
In fact, it is complete random. The original style was just recording the json and storing it as just regular python way that cause random order for each records but the list usually comes first. And affiliation is a list so it might usually comes first. Let's create an issue to handle this.